website/static/js/pages/project-settings-page.js

```diff
@@ -252,5 +252,57 @@ $(document).ready(function() {
             }
         }
     });
+    $('.wiki-select').on('change', function() {
+        var self = this;
+        var $self = $(self);
+        var wikiEnabled = $self.is(':checked');
```

Comment:
Could you simplify this and prevent storing state in the DOM by using a simple ViewModel? You could even use a POJO:

```js
var WikiSettingsViewModel = {
     enabled: ko.observable(ctx.wiki.isEnabled), // <- this would get set in the mako template, as usual
     onChange: function(e) { 
         // send request then reload page
      }
}
```

```html
...
<input type="checkbox" data-bind="event: onChange, checked: enabled" />
```